test/cql-pytest/test_allow_filtering.py

```diff
@@ -175,7 +174,7 @@ def wait_for_index(cql, table, column, everything):
         results = []
         for v in column_values:
             results.extend(list(cql.execute(f'SELECT * FROM {table} WHERE {column}={v}')))
-        if set(results) == set(everything):
+        if sorted(results) == sorted(everything):
```

Comment:
I'm curious why this change was needed. Is it because you wanted to properly support duplicates or something? sorted with items that can be sorted. In test/cql-pytest/cassandra_tests/porting.py I have multiset that I used for similar things for more elborate types - and can also be moved to util.py (in test/alternator.py I also had it).

Reply:
One of the new queries returns a list, but python refuses to do set of lists because `list is not hashable`
afaik python doesn't have a `TreeSet` in its standard library

Doing `sorted` was a quick fix. I convinced myself that duplicates wouldn't be an issue because column_values return rows from different partitions each time. There is probably a better and cleaner way to do this.

Reply:
I changed this to something much simpler and less error prone. There are very few rows so it should be fast enough.